### DISCUSSION TITLE
How long should I wait for a response?

### DISCUSSION THREAD

**Discussion by mxmnk** [community/community#22412] [New to GitHub]
<p>Hello! My account got flagged, so I wrote to private GitHub Support. I have been waiting for 2 weeks and still have not received a response.</p>
<p>How long should I wait for a response? My Ticket ID: 933655</p>
<p>Thanks in advance :slight_smile:</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>The wait depends on why your account was flagged, and whether the appeal requires review from one team, or many.</p>
<p>Two weeks does seem like a lot. I’ll flag your ticket just in case it got missed!</p>

**Comment by canuckjacq** [comment]
<p>4 posts were merged into an existing topic: <a href="https://github.community/t/github-student-pack-status-check/117951/13">GitHub Student Pack Status Check</a></p>

**Comment by milfan** [comment]
<p>Hi. Can you help my ticket too ? I have been waiting for 8 days and no received any response. Thank you</p>

**Comment by nasserlouhichi** [comment]
I am also still waiting can you for godsake unflagg my account it was created 2 days ago and I have linked it to digital ocean and now I lost the access I haven't do anything wrong for godsake

**Comment by baharbaba** [comment]
I've been waiting for a response for 1 month. A week ago, I received a message saying "We have not forgotten about you..." but now I feel like I have been forgotten. From now on, I'll be tagging @github-staff in every discussion that has this issue. I guess you need to be Karen to get a problem solved.
